services/bstats/bstats.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const Joi = require('joi')
+
+const ServiceTester = require('../service-tester')
+const t = (module.exports = new ServiceTester({
+  id: 'bstats',
+  title: 'bStats badges',
+}))
+
+const withRegex = re => Joi.string().regex(re)
+
+const isNumber = withRegex(/^[-+]?\d+$/)
```

Comment:
Does the underlying response from the bStats API include an optional `-` or `+` prefix? Or is it just a number? From what I've read it (albeit briefly) it looks to just be integers, in which case you could just use `Joi.number()` for the value checks on lines 21 and 31 right?